02-express-tutorial/routes/products.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+
+const { getProducts, getSingleProduct } = require("../controllers/products");
+
+const router = express.Router();
+
+console.log("router object BEFORE adding routes =", router);
+
+// products end points
+// /api/v1/products requests
+router.route("/").get(getProducts);
```

Comment:
The route here should be "/products".

Reply:
Hey Luke! I just wanted to thank you for your time and I am loving the feedback! I wanted to make sure that I am not going about this wrong. I believe the route that is being required of us is '/api/v1' + '/people' and so on for the other end points below. I have provided a screenshot for context.

I could also be misunderstanding but I sectioned off the end points in app.js on lines 36-42. I did it this way, so I would not have to include "/people", "products", etc. in the routes folder. Let me know if this is okay and if not I'll make the necessary changes. Thanks!

![Snag_14b841b5](https://github.com/Bungycode/node-express-course/assets/84283930/f1c111eb-f1db-4ccf-8b4c-7aaee48708ab)